### ISSUE TITLE
Bug Report - Titlebar bug

### ISSUE THREAD

**Issue by JushBJJ** [microsoft/terminal#1539]
# Environment

```none
Windows build number: [10.0.18362.175]
Windows Terminal version (if applicable): 0.2.1715.0

Any other software?
Minecraft (This bug showed up when I opened up this application, but can't confirm if that's the cause.)
Chromium Edge [Dev] (3 tabs open)
```

# Steps to reproduce

I'm clearly unsure how I reproduced this but this appeared when I opened up Minecraft, moved to Desktop 2 (using trackpad) then moved back to Desktop 1. The transition of moving was quite laggy. Also, one of the tabs in Chromium Edge [Dev] was loading while Minecraft was starting up. Then after, re-opened the Terminal window (it was minimized before).
# Expected behavior

Title bar shows the modern windows 10 titlebar design.
# Actual behavior
(The obvious red lines are to be ignored, the red arrow points to the issue)

![image](https://user-images.githubusercontent.com/36951064/60030157-d400a500-96e5-11e9-9a9e-37e64128bf90.png)

**Comment by JushBJJ** [comment]
Update: The titlebar fixes when the window is maximized but breaks again when the floating.

**Comment by DHowett-MSFT** [comment]
Wow! This one we haven't seen before. Thanks.

**Comment by DHowett-MSFT** [comment]
It will probably be addressed by #929 and #872

**Comment by JushBJJ** [comment]
Oh I see, I hope this gets fixed soon!
